README.rst

```diff
@@ -387,17 +190,92 @@ More examples shown below.
     and it is dependent on the tool being used.
 
 
+POST /cbd_decrypt
```

Comment:
i'd be keen to have this as similar to the `nucypher-ts` api as possible. So in `ts` where we have `taco.decrypt` it should probably be `taco_decrypt` here

Reply:
👍 - I'm good with that name change unless someone else objects (cc @cygnusv , @KPrasch , @arjunhassard ).

That being said, I'd like to do it in a separate PR, since I'll also have to make changes to `nucypher-ts`. Filed #42